sql/opt_range.cc

```diff
@@ -5757,7 +5757,7 @@ bool prepare_search_best_index_intersect(PARAM *param,
     {
       idx_scan.add("chosen", true);
       if (!*scan_ptr)
-        idx_scan.add("cause", "first occurence of index prefix");
+        idx_scan.add("cause", "first occurrence of index prefix");
```

Comment:
Now this is an interesting find!

No test cases for this code. We should have seen this change reflect as a test case failure.

Ok to push however. Writing a (stable) test case for this is not trivial.